.circleci/config.yml

```diff
@@ -368,6 +416,7 @@ workflows:
             - build
       - deploy:
           requires:
+            - build
```

Comment:
Why did we add build explicitly here? If build fails all the rest of the steps wont run either.

Reply:
Whoops- definitely a copy paste error there. I keep removing and adding back the required jobs for deploy so I can test out deploys faster on my feature branches. I'm working on a PR to merge into josh's https://github.com/adhocteam/Head-Start-TTADP/pull/78 (I'm correcting the seeding that runs on CD). I can fix this then, or just open a new simple PR if that's better. Let me know.

Reply:
Can we remove it here and also switch over to using `DATABASE_URL` instead of all of the CircleCI configured POSTGRES variables?